pkg/cloudprovider/providers/gce/cloud/gen/main.go

```diff
@@ -0,0 +1,1142 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Generator for GCE compute wrapper code. You must regenerate the code after
+// modifying this file:
+//
+//   $ go run gen/main.go > gen.go
```

Comment:
Maybe adding a script (`hack/verify-xxx.sh`) to ensure the generated code is in sync with the committed input/generation code?